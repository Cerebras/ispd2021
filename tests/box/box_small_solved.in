2
6 6
1
5 5
# this is a comment
0.5 0.5
# y = 0
0.5
0.5
0.5
0.5
0.5
0.5
0.5
# y = 1
0.5
0.5
0.5
0.5
0.5
0.5
0.5
# y = 2
0.5
0.5
1
1 
1
0.5
0.5
# y = 3
0.5
0.5
1 
1
1
0.5
0.5
# y = 4
0.5
0.5
1 
1
1
0.5
0.5
# y = 5
0.5
0.5
0.5
0.5
0.5
0.5
0.5
# y = 6
0.5
0.5
0.5
0.5
0.5
0.5
0.5