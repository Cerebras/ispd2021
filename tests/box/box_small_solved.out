1
1 0 0 3 1
1 0 2 1 1
1 4 2 1 1
1 0 4 3 1
0 2 2 2 2
compute_map:
0 0
0 1
0 2
0 3
1 0
1 1
1 2
1 3
2 0
2 1
2 2
2 3
adapter_map:
3 0
3 1
3 2
3 3
3 4
0 4
1 4
2 4